lms/djangoapps/student_account/views.py

```diff
@@ -267,7 +267,8 @@ def enterprise_sidebar_context(request):
 
     platform_name = configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)
 
-    logo_url = enterprise_customer.get('branding_configuration', {}).get('logo', '')
+    branding_configuration = enterprise_customer.get('branding_configuration', {})
+    logo_url = branding_configuration.get('logo', '') if isinstance(branding_configuration, dict) else ''
```

Comment:
Hmm, why would the branding configuration not be a `dict`?

Reply:
@UmanShahzad ^^

Reply:
@douglashall If it's not set, it seems to come in as a `None`. See for example on my devstack:

```json
{  
   "uuid":"185938c8-7a19-4995-b0b0-3f0ad5fb5f3a",
   "name":"Development Enterprise",
   "catalog":1,
   "active":true,
   "site":{  
      "domain":"example.com",
      "name":"example.com"
   },
   "enable_data_sharing_consent":true,
   "enforce_data_sharing_consent":"at_enrollment",
   "branding_configuration":null,
   "enterprise_customer_entitlements":[  
   ],
   "enable_audit_enrollment":true
}
```

Since it still exists in the response though, the `enterprise_customer.get` function here just returns `None` rather than the dict, and that's the bug: the expectation apparently was that even if it returns `None`, it'll go to the default of `{}`, but it doesn't work that way.

Reply:
Got it. Thanks.